wallet-new/server/Cardano/Wallet/Server/Plugins.hs

```diff
@@ -89,17 +93,18 @@ legacyWalletBackend :: (HasConfigurations, HasCompileInfo)
                     -> Plugin WalletWebMode
 legacyWalletBackend WalletBackendParams {..} ntpStatus =
     first one $ worker walletServerOuts $ \diffusion -> do
-      logInfo $ sformat ("Production mode for API: "%build)
-        walletProductionApi
-      logInfo $ sformat ("Transaction submission disabled: "%build)
-        walletTxCreationDisabled
-
-      walletServeImpl
-        (getApplication diffusion)
-        walletAddress
-        -- Disable TLS if in debug mode.
-        (if isDebugMode walletRunMode then Nothing else walletTLSParams)
-        (Just $ setOnExceptionResponse exceptionHandler defaultSettings)
+      modifyLoggerName (const "legacyServantBackend") $ do
```

Comment:
Here I have simply added this custom logger name. You will notice (here and in the rest of the `modifyLoggerName` calls) that I am using `const` and not `mappend`. This is intentional as I have discovered that `log-warper` couldn't be configured (at least, I  failed in doing so) to accept a composite logger name (like `foo.bar.baz`) in the `yaml` config file to set the verbosity for a [specific logger](https://github.com/serokell/log-warper/blob/master/examples/how-to-log-config.yaml). This helped my testing quite a lot, but I'm happy to revert.